ci/Jenkinsfile.android

```diff
@@ -24,6 +24,7 @@ pipeline {
 
   environment {
     BUILD_PLATFORM = 'android'
+    NODE_VERSION = 'v10.14.1'
```

Comment:
This should be `10.12.0`, right?

Reply:
would be cool if we could read the version from `.TOOLVERSIONS` :-)

Reply:
Oh, is that change already merged? Then i can try that.